website/docusaurus.config-blog.js

```diff
@@ -50,6 +50,26 @@ const config = {
 
           blogSidebarTitle: 'All posts',
           blogSidebarCount: 'ALL',
+          feedOptions: {
+            type: 'all',
+            title: 'The FerretDB Blog',
+            description: 'Welcome to the FerretDB blog - The truly open source replacement for MongoDB',
+            copyright: `Copyright © ${new Date().getFullYear()} FerretDB, Inc.`,
+            createFeedItems: async (params) => {
+              const {
+                  blogPosts,
+              } = params;
+
+              return blogPosts.slice(0, 10).map(post => ({
+                  title: post.metadata.title,
+                  link: post.metadata.frontMatter.link,
+                  // appending the siteUrl to the img src as a workaround for the relative paths used for images
+                  image: `https://blog.ferretdb.io${post.metadata.frontMatter.image}`,
```

Comment:
It's a Docusaurus issue and you can take a look at this discussion here –https://github.com/facebook/docusaurus/discussions/8321#discussioncomment-7016367.

This is a screenshot of the RSS feed (Docusaurus default) currently and there's no tag for the blog post images. 

<img width="1011" alt="Screenshot 2023-09-25 at 23 41 02" src="https://github.com/FerretDB/FerretDB/assets/24963785/fed6b352-75bc-4cd5-92d1-0ce3a72ba340">

> but you are not appending `siteUrl` – you are appending the fixed string that is not valid for previews, for example. That makes it impossible to validate feeds on the preview environment.

> But they are not relative in the feed – they are absolute.
> 
> Was that code copied from somewhere? Is there a known issue in Docusaurus?

No, it wasn't copied. It was done as a workaround to get the images to appear and also customize the feeds themselves. I'm open to any suggestions that could work, otherwise, we'll have to manually enter each latest blog post on the website instead of getting them automatically via the RSS feed.

Reply:
The new FerretDB website being designed by Peter Benei.

Reply:
I just pushed a small clean-up. The problem with the current `createFeedItems` replacement is that it removes the content, keeping only a short description. I don't think we should do that.

The default function is there: https://github.com/facebook/docusaurus/blob/ae4a6f9d949619d492b69114213fd32786ac48f7/packages/docusaurus-plugin-content-blog/src/feed.ts#L79 Maybe we could adopt it somehow.

Reply:
Thanks! I'll take a look and add the `content` for the feed to it too.